geode-core/src/test/java/org/apache/geode/test/junit/assertions/GfshShellConnectionRuleAssert.java

```diff
@@ -208,6 +208,31 @@ public GfshShellConnectionRuleAssert tableHasColumnWithValuesContaining(String h
     return this;
   }
 
+  public GfshShellConnectionRuleAssert tableHasColumnWithValueMatchingOneOf(String header,
+      String... acceptedValues) {
+    GfJsonObject resultContentJSON = actual.getCommandResult().getContent();
+    Object content = resultContentJSON.get(header);
+
+    if (content == null) {
+      failWithMessage("Command result did not contain a table with column header <" + header + ">: "
+          + resultContentJSON.toString());
+    }
+
+    Object[] actualValues = toArray((JSONArray) content);
+
+    for (Object actualValue : actualValues) {
+      String actualValueString = (String) actualValue;
+      boolean actualValueContainsAnAcceptedValue =
+          Arrays.stream(acceptedValues).anyMatch(actualValueString::contains);
+
+      if (actualValueContainsAnAcceptedValue) {
+        return this;
+      }
+    }
+    failWithMessage("No accepted value found.");
+    return null;
```

Comment:
I think you can still return `this` rather than returning `null`.  (For instance, someone might be using a JUnit rule that gathers up exceptions and re-throws them at the end of a sequence of operations.  We wouldn't want to give them a NPE here in that case.)

Reply:
I hadn't even considered that scenario.  Done and done.